include/verilated_types.h

```diff
@@ -415,8 +415,20 @@ class VlWriteMem final {
 
 static int _vl_cmp_w(int words, WDataInP const lwp, WDataInP const rwp) VL_PURE;
 
+template <std::size_t T_Words>
+struct VlWide;
+
+// Type trait to check if a type is VlWide
+template <typename>
+struct IsVlWide : public std::false_type {};
```

Comment:
```suggestion
struct VlIsWide : public std::false_type {};
```

Or VlIsVlWide.  I'd like to keep all functions etc in here starting with VL as there's no namespace around them.